stdlib/tkinter/ttk.pyi

```diff
@@ -293,8 +293,8 @@ class Combobox(Entry):
     ) -> dict[str, tuple[str, str, str, Any, Any]] | None: ...
     @overload
     def config(self, cnf: str) -> tuple[str, str, str, Any, Any]: ...
-    def current(self, newindex: Any | None = ...): ...
-    def set(self, value): ...
+    def current(self, newindex: int | str | None = ...) -> int: ...
```

Comment:
Shouldn't allow `newindex` to be string, and returns empty string when `newindex` is specified as an `int`.

```suggestion
    @overload
    def current(self, newindex: None = ...) -> int: ...
    @overload
    def current(self, newindex: int) -> Literal[""]: ...
```